ecmd-core/ext/fapi2/capi/fapi2_target.H

```diff
@@ -35,8 +35,6 @@
 #include <plat_target.H>
 #include <return_code_defs.H>
 
-#define FAPI2_ENABLE_PLATFORM_GET_CORE_SELECT
```

Comment:
Why did this get removed?

Reply:
it should instead be defined in a makefile for the platform and is not needed in header file